lib/index.js

```diff
@@ -61,13 +62,19 @@ HubSpot.prototype.page = function() {
  */
 
 HubSpot.prototype.identify = function(identify) {
-  if (!identify.email()) return;
-  var traits = identify.traits({ firstName: 'firstname', lastName: 'lastname' });
+  // use newer version of Identify to have access to `companyName`
+  var newIdentify = new Identify({ traits: identify.traits() });
```

Comment:
i think you also want to pass in `userId: identify.userId()` along with traits here since userId is top level and not usually nested inside traits and it takes userId as a param: https://github.com/segmentio/facade/blob/master/lib/identify.js#L12-L24

so: `new Identify({ userId: identify.userId(), traits: identify.traits() })